addons/FlashCards/src/presenter.js

```diff
@@ -386,6 +386,7 @@ function AddonFlashCards_create(){
         return !$(presenter.$view.find(".flashcards-card").get(0)).hasClass("flashcards-card-reversed");
     }
 
+    presenter.originalCard = -1;

```

Comment:
Karta numer 1 jest na początku odkryta, a ta zmienna jest wykorzystywana tylko przy presenter.showCard i presenter.nextCard. Biorąc pod uwagę, że ta zmienna ma śledzić indeksy to presenter.originalCard powinno być równe 1. 

Możliwe, że dla obecnej konfiguracji nie jest to zauważalne bo w presenter.init jest wykonywane presenter.showCard, gdzie presenter.originalCard według kodu jest ustawiane na 1.

Proponuję tą zmianę, aby wartość zgadzała się z początkową wartością presenter.state.currentCard, a tym samym ujednolicając kod.